cms/envs/aws.py

```diff
@@ -57,8 +57,8 @@
 BROKER_POOL_LIMIT = 0
 BROKER_CONNECTION_TIMEOUT = 1
 
-# For the Result Store, use the django cache named 'celery'
-CELERY_RESULT_BACKEND = 'djcelery.backends.cache:CacheBackend'
+CELERY_RESULT_BACKEND = 'cache'
+CELERY_CACHE_BACKEND = 'memcached://127.0.0.1:11211/'
```

Comment:
According to the docs, as of celery 3.1 (we seem to be pinned at 3.1.18 here), this second setting is no longer used and has been merged into the first: http://docs.celeryproject.org/en/3.1/configuration.html#cache-backend-settings.  Not sure how accurate that is, given that it also uses the supposedly abandoned setting in one of the examples...